### DISCUSSION TITLE
Can LogAPI be bundled with Sandboxie-Plus?

### DISCUSSION THREAD

**Discussion by bastik-1001** [sandboxie-plus/Sandboxie#2161] [Various ideas and suggestions] [answered]
Is there a reason why LogAPI is not bundled with the installer of Sandboxie-Plus?

It shouldn't be much overhead at all, to provide the dlls in the installer. At least for the plus version.

Now that I ask about why it isn't bundled with the installer, a follow-up question came up, why isn't that integrated into Sandboxie, anyway?

**Comment by DavidXanatos** [comment, marked as answer]
Well the LogAPI dll is not really elegant code, and it is missing many api hooks, so its ugly and incomplete.

Also sandboxie now has the ability to log all syscalls that should be even more verbose anyways
